specs/latest/2.0/index.html

```diff
@@ -3344,6 +3563,25 @@ <h2 class="no-toc">Table of contents</h2>
         this limitation in WebGL 2 for all platforms.
     </div>
 
+    <h3><a name="DOM_UPLOAD_UNPACK_PARAMS">Pixel store parameters for uploads from
+        <code>TexImageSource</code></a></h3>
+    <p><code>UNPACK_ALIGNMENT</code> and <code>UNPACK_ROW_LENGTH</code> are ignored, as they
+       determine row-stride-in-bytes, which is immutable for TexImageSource objects.
+    <p>Subrect selection is possible using <code>UNPACK_</code> params.
+       <code>UNPACK_SKIP_PIXELS</code> and <code>UNPACK_SKIP_ROWS</code> determine the origin of the
+       subrect, with <code>width</code> and <code>height</code> determining the size of the subrect.
```

Comment:
Here you need to clarify what happens when the sub-region is beyond the source, i.e., UNPACK_SKIP_PIXELSS + width > source_width or UNPACK_SKIP_ROWS + height > source_height. We can either fill with zeros or generate an INVALID_OPERATION.  I prefer the latter for simpler implementation.